[2022-02-01 07:23:44,234] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_file_changes scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-01 07:23:44,280] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_file_changes scheduled__2022-01-31T00:00:00+00:00 [queued]>
[2022-02-01 07:23:44,281] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-02-01 07:23:44,282] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2022-02-01 07:23:44,283] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-02-01 07:23:44,335] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): check_file_changes> on 2022-01-31 00:00:00+00:00
[2022-02-01 07:23:44,357] {standard_task_runner.py:52} INFO - Started process 92 to run task
[2022-02-01 07:23:44,382] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'check_file_changes', 'scheduled__2022-01-31T00:00:00+00:00', '--job-id', '3140', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpu1imdf2u', '--error-file', '/tmp/tmp1kvhybwq']
[2022-02-01 07:23:44,384] {standard_task_runner.py:77} INFO - Job 3140: Subtask check_file_changes
[2022-02-01 07:23:44,564] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.check_file_changes scheduled__2022-01-31T00:00:00+00:00 [running]> on host 218df5ddc363
[2022-02-01 07:23:44,794] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=check_file_changes
AIRFLOW_CTX_EXECUTION_DATE=2022-01-31T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-31T00:00:00+00:00
[2022-02-01 07:23:48,590] {python.py:152} INFO - Done. Returned value was: None
[2022-02-01 07:23:48,784] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=check_file_changes, execution_date=20220131T000000, start_date=20220201T072344, end_date=20220201T072348
[2022-02-01 07:23:49,012] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-01 07:23:49,323] {local_task_job.py:264} INFO - 3 downstream tasks scheduled from follow-on schedule check
